src/mds/tests/rolling_test_base.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+
+from mds.mrs.models import BinaryResource, Client, SavedProcedure
+
+import os
+from shutil import rmtree
+import uuid
+
+class RollingTestCase(TestCase):
+    def setUp(self):
+        self.block_size = 1024
```

Comment:
nit: we can define this only once, as opposed to for every test(), unless you anticipate a test changing this